judgments/tests.py

```diff
@@ -396,13 +396,14 @@ def test_min_max():
 
 
 def test_prep_query():
-    assert utils.remove_unquoted_stop_words("weight of evidence") == "weight evidence"
-    assert (
-        utils.remove_unquoted_stop_words("'weight of evidence'")
-        == "'weight of evidence'"
-    )
+    # Stopwords outside quoted strings are removed.
+    assert utils.preprocess_query("weight of evidence") == "weight evidence"
```

Comment:
Could be worth splitting each of these asserts into their own test functions since they are testing different aspects of the functions and if they are so.

e.g. 
 # Stopwords outside quoted strings are removed.
    assert utils.preprocess_query("weight of evidence") == "weight evidence"
    
    can become `test_preprocess_query_removes_stopwords_quoted_strings`


At minimum I think we should definitely split out the pre_process_query asserts from the remove_unquoted_stop_words asserts.

Reply:
Agree that `preprocess_query` and `remove_unquoted_stop_words` are two different things. I'm not sure each assertion needs an individual test at this point unless we see regressions in future though, but I have no strong opinions either way so if we want the robustness then go for it!